QUEST_en.tsv

```diff
@@ -13,17 +13,17 @@ QUEST_20150317_000012	Bokor Master
 QUEST_20150317_000013	You want to know about this slate? {Nl} I'll show you the vision of the one who made this slate
 QUEST_20150317_000014	Goddess Laima
 QUEST_20150317_000015	By the time you see this revelation, a long time must have passed by. {Nl} I am Laima, the Goddess of Fate and Luck. {Nl}
-QUEST_20150317_000016	When I made this revelation to 600 years ago, I saw a lot of grief and death. {Nl} Your seeing this revelation would mean that all I predicted have become reality. {Nl}
+QUEST_20150317_000016	When I made this revelation 600 years ago, I saw a lot of grief and death. {Nl} Your seeing this revelation would mean that all I predicted have become reality. {Nl}
 QUEST_20150317_000017	The first disaster was cause by the Demons, and all the Goddesses may have disappeared. {Nl} I know all of this beforehand and tried my best to prevent it from happening, but my power was not enough. {Nl}
-QUEST_20150317_000018	I walked through the long path of predictions to find the answer. {Nl} At the end of the dream was you.. The Savior. {Nl}
+QUEST_20150317_000018	I walked through the long path of predictions to find the answer. {Nl} At the end of the dream, it was you.. The Savior. {Nl}
 QUEST_20150317_000019	However, knowing my power to foresee fate, the Demons would not have left me alone {nl}so I had to hide myself until I meet you. {Nl}
-QUEST_20150317_000020	I decided to divide and put myself in multiple of revelations. {Nl} Making and hiding myself in revelations around the kingdom continued for a thousand years. {Nl}
+QUEST_20150317_000020	I decided to divide and put myself in multiple revelations. {Nl} Making and hiding myself in revelations around the kingdom continued for a thousand years. {Nl}
 QUEST_20150317_000021	Dear Savior. {Nl} In order for me to stop the Demons, {nl}you need to find all the revelations that I have hidden in the Axis of Time.
 QUEST_20150317_000022	The burden of the sacred mission could press down your shoulders. {nl} . {Nl}
 QUEST_20150317_000023	But, only you and I could save the world.{nl}
 QUEST_20150317_000024	I passed the next revelation to the first Paladin.{nl}I will be waiting for you at the garden at the top.
 QUEST_20150317_000025	Evil forces and will be but I do not feel at all. {Nl} Then, this revelation is true of Laima goddess .. just awesome.
-QUEST_20150317_000026	Okay, Please go back to Uska.{nl}He will tell you where to go.{nl}May the Goddess bless for your future..
+QUEST_20150317_000026	Okay, Please go back to Uska.{nl}He will tell you where to go.{nl}May the Goddess bless you for your future..
```

Comment:
It is more correct to remove "for" instead of adding "you" so it reads "May the Goddess bless your future."